### PR TITLE
[HOPSWORKS-1372] Upgrade the ELK stack to 7.2 OSS and use the OpenDis…

### PR DESCRIPTION
**Pull request by maismail**
…tro Security Plugin
https://logicalclocks.atlassian.net/browse/HOPSWORKS-1372
tests: https://github.com/logicalclocks/hops-testing/pull/565

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -1334,7 +1334,57 @@
           :description =>  "Sha-1 checksum for the elasticsearch .tar.gz file",
           :type => 'string'
 
+attribute "elastic/opendistro_security/admin/username",
```
Comment (SirOibaf) [inline metadata.rb]: There is no need to redefine all these attributes here - Since version 0.4 Karamel also checks the `metadata.rb` of the dependent cookbooks

File: recipes/default.rb [hunk 2]
```diff
@@ -119,7 +119,7 @@
 end
 
 begin
-  kibana_ip = private_recipe_ip("hopslog","default")
+  kibana_ip = public_recipe_ip("hopslog","default")
```
Comment (SirOibaf) [inline recipes/default.rb]: What is the reason for this change? We use everywhere the `private_recipe_ip`.

File: metadata.rb (hunk 1 above)
Comment (maismail) [reply metadata.rb]: Ok, good to know. I just added it here since i found that we are doing this for other cookbooks.

File: recipes/default.rb (hunk 2 above)
Comment (maismail) [reply recipes/default.rb]: Since we are removing the kibana proxy servlet and instead exposing a secure kibana to users.
